runtime/development/src/lib.rs

```diff
@@ -1674,6 +1708,27 @@ impl pallet_liquidity_rewards::Config for Runtime {
 	type WeightInfo = ();
 }
 
+frame_support::parameter_types! {
+	pub const BlockRewardsDomain: RewardDomain = RewardDomain::Block;
+	pub const BlockRewardCurrency: CurrencyId = CurrencyId::Staking(BlockRewardsCurrency);
+}
+
+impl pallet_block_rewards::Config for Runtime {
+	type AdminOrigin = EnsureRootOr<HalfOfCouncil>;
+	type AuthorityId = AuraId;
+	type Balance = Balance;
+	type Beneficiary = ();
```

Comment:
Since Centrifuge runtime will have the treasury here as the beneficiary, would it make sense to already add it here, so we can verify that piece works fine?

Reply:
Good catch! [5be160c](https://github.com/centrifuge/centrifuge-chain/pull/1198/commits/5be160c8690ecda671f2aefa4615c6bda15d9ddd)